index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body class="container">
+    <div id="boxOne" class ="boxOne">
+    <h1 id="headerOne" class="headerOne">My name is Dan!</h1>
+    <p>My first name is Dan. And last name is Bennington. I think the picture of the dog is hilarious. 
+        I was going to write a bunch of text ... but I ran into time constraints. Woohoo! I was hoping 
+        to write a bunch of text, almost a novel. But of course, the clock is ticking and I'm getting down
+        to the wire.
+    </p><br>
+        <img id="imgOne" class="imgOne" src="https://i.ytimg.com/vi/zfBG0d5Oj3c/hqdefault.jpg" alt="Dog caught his tail">
+        <section class="move"><br>
+            <h2 id="headerTwo" class="headerTwo">A quick list of likes:</h2>
+                <ol id="listOne" class="listOne">
+                    <li>Food!</li>
+                    <br>
```

Comment:
in general you'll want to do spacing like this via CSS and not the old-school br tag